docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device/batching-on-npu-plugin.rst

```diff
@@ -0,0 +1,33 @@
+NPU Plugin batching
+===============================
+
+
+.. meta::
+   :description: The Bathing is handled on the NPU plugin in OpenVINO™
+                 in two different modes, concurrency-based inferences
+                 or handled by the compiler.
+
```

Comment:
In OpenVINO™, the NPU plugin handles batching in two modes: concurrency-based inference and compiler-handled execution.

Reply:
let's maybe add it or smth similar to the beginning of the page? since the description will be hidden

Reply:
done, thx